Spring-server/src/main/java/com/controller/StudentController.java

```diff
@@ -0,0 +1,55 @@
+package com.controller;
+
+import com.model.Student;
+import com.repository.StudentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping(value = "/api/")
+public class StudentController {
+    @Autowired
+    private StudentRepository studentRepository;
+    @GetMapping("/students")
+    public List<Student> getUsers(){
```

Comment:
Controller layer should never accept or return database entities. We have to use DTOs instead

Reply:
Moreover, DTO classes should be validated using tools provided by Spring.